unit_test/test_size/test_size_of_spdm_responder/CMakeLists.txt

```diff
@@ -43,6 +44,8 @@ SET(test_size_of_spdm_responder_LIBRARY
     spdm_secured_message_lib
     spdm_transport_mctp_lib
     spdm_device_secret_lib_null
+#    spdm_device_secret_lib_sample
```

Comment:
Delete the commented lines.

Reply:
I leave this purposely, because I can enable real sample lib for size collection.